src/components/loading/loading_elastic.test.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { render, mount } from 'enzyme';
+import { requiredProps } from '../../test';
+import { getDeprecatedMessage } from '../../utils';
+import { OuiLoadingElastic, SIZES, WARNING } from './loading_elastic';
+
+describe('OuiLoadingElastic', () => {
+  test('is rendered', () => {
+    const component = render(<OuiLoadingElastic {...requiredProps} />);
+
+    expect(component).toMatchSnapshot();
+  });
+
+  describe('size', () => {
+    SIZES.forEach((size) => {
+      test(`${size} is rendered`, () => {
+        const component = render(<OuiLoadingElastic size={size} />);
```

Comment:
Should this also get `{...requiredProps}`?

Reply:
Yes, thank you for noticing that!